packages/kbn-test/src/functional_tests/lib/run_elasticsearch.ts

```diff
@@ -79,7 +79,11 @@ export async function runElasticsearch(
   options: RunElasticsearchOptions
 ): Promise<() => Promise<void>> {
   const { log, logsDir, name } = options;
-  const config = getEsConfig(options);
+  let config = getEsConfig(options);
+  config = {
+    ...config,
+    esArgs: ['xpack.ml.enabled=false', ...config.esArgs],
```

Comment:
It might be a breaking change for other tests, could you pass `'xpack.ml.enabled=false'` explicitly in your FTR config?

Reply:
@dmlemeshko, yeah sorry; I rolled back this change already